src/DebugApi.ts

```diff
@@ -281,7 +281,7 @@ export class DebugApi {
         });
     }
 
-    private async killPortal (req: IncomingMessage, response: ServerResponse) {
+    private async killRoom (req: IncomingMessage, response: ServerResponse) {
```

Comment:
This looks very backwards incompatible. Was the `killPortal` undocumented and thus can just be replaced? Or are we fine because it's in the debug API?

Reply:
The latter. It's called out in the changelog, and we don't make any guarantees that the debugAPI won't change.